custom_components/tahoma/climate_seh.py

```diff
@@ -0,0 +1,68 @@
+"""Support for Atlantic Electrical Heater IO controller."""
+import logging
+from typing import List
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import HVAC_MODE_HEAT, HVAC_MODE_OFF
+from homeassistant.const import TEMP_CELSIUS
+
+from .tahoma_device import TahomaDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+COMMAND_DOWN = "down"
+COMMAND_MY = "my"
+COMMAND_OFF = "off"
+COMMAND_ON = "on"
+COMMAND_UP = "up"
+
+HVAC_MODE_UNKNOWN = "Unknown"
+
+PRESET_UNKNOWN = "Unknown"
+
+
+class StatelessExteriorHeating(TahomaDevice, ClimateEntity):
+    """Representation of TaHoma IO Atlantic Electrical Heater."""
+
+    def __init__(self, tahoma_device, controller):
+        """Init method."""
+        super().__init__(tahoma_device, controller)
+
+    @property
+    def supported_features(self) -> int:
+        """Return the list of supported features."""
+        return 0
+
+    @property
+    def temperature_unit(self) -> str:
+        """Return the unit of measurement used by the platform."""
+        return TEMP_CELSIUS
+
+    @property
+    def hvac_mode(self) -> str:
+        """Return hvac operation ie. heat, cool mode."""
+        return HVAC_MODE_UNKNOWN
```

Comment:
What if you return `None`? Normally you don't return a string, but `None` will default to `Unknown`.

Reply:
I did this to have a `str` but if `None` is fine, that would be better!

Reply:
changed to None